apps/api/v2/src/ee/me/me.controller.ts

```diff
@@ -60,18 +63,65 @@ export class MeController {
     @GetUser() user: UserWithProfile,
     @Body() bodySchedule: UpdateManagedUserInput
   ): Promise<UpdateMeOutput> {
+    const emailVerification = await this.featuresRepository.checkIfFeatureIsEnabledGlobally(
```

Comment:
Controllers should only receive request, extract necessary data from requests like authenticated user and pass information to service and then service responds and controller's second responsibility is to then craft a response.

Given that, can we please create a new `me.service.ts` with a function `updateMe` which will all of the logic in this controller.